pgjdbc/src/test/java/org/postgresql/test/ssl/ClientRevocationListTest.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2018, PostgreSQL Global Development Group
+ * See the LICENSE file in the project root for more information.
+ */
+
+package org.postgresql.test.ssl;
+
+import org.postgresql.PGProperty;
+import org.postgresql.jdbc.SslMode;
+import org.postgresql.test.TestUtil;
+import org.postgresql.util.PSQLException;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.File;
+import java.sql.Connection;
+
+import java.util.Properties;
+
+public class ClientRevocationListTest {
```

Comment:
A test must be added to a suite. Otherwise it is not executed

Reply:
Yes, I remembered that after I pushed it